### ISSUE TITLE
Menu Components | Cannot override model value

### ISSUE THREAD

**Issue by cetincakiroglu** [primefaces/primeng#13471]
All menu components have the same defect after the accessibility update (16.1.0). Users cannot override model after initial value. Components do not respond whether model value changed or not.

Components to be fixed:

- Contextmenu
- MegaMenu
- MenuBar
- SlideMenu
- TabMenu
- TieredMenu

**Comment by Github743** [comment]
Is there any fix available for this, I had similar issue can not perform an action when we add items on an event instead of ngOnInit https://stackoverflow.com/questions/77998575/primeng-menuitem-with-command-not-working
